apiserver/cmd/server/server.go

```diff
@@ -2999,12 +3023,13 @@ func (s *Server) GetHealthz(w rest.ResponseWriter, r *rest.Request) {
 		return
 	}
 
+	// TODO: Remove this once PVCs are supported?
```

Comment:
Should be removed with this PR along with getGlusterStatus and updateStorageQuota methods